news/2025/2025-08-22-beatmap-spotlights-summer-2025.md

```diff
@@ -0,0 +1,216 @@
+- [Hagumi Nishizawa - My Hero! Up to you! (Game Ver.) (Katsuragi) \[Rain\]](https://osu.ppy.sh/beatmapsets/2239390#osu/4758983), suggested by [Morusya](https://osu.ppy.sh/users/13681464)
+- [-45 - System Zero (skill issue lol) \[2zz's Rain\]](https://osu.ppy.sh/beatmapsets/1984907#osu/4566061), suggested by [Morusya](https://osu.ppy.sh/users/13681464)
+- [IOSYS - Chanteikku Sanyousei no Itazura Daisensou (Spectator) \[Rocma's Rain\]](https://osu.ppy.sh/beatmapsets/2281583#osu/4879141), suggested by [Morusya](https://osu.ppy.sh/users/13681464)
+- [Azuna Riko - Katachi (TV Size) (Mniam) \[Faputa (CV: Mniam)\]](https://osu.ppy.sh/beatmapsets/1925863#osu/3976666), suggested by [Phob](https://osu.ppy.sh/users/6069462)
+- [Sasuke Haraguchi feat. Kasane Teto - Igaku (Spectator) \[Overdose\]](https://osu.ppy.sh/beatmapsets/2230485#osu/4733442), suggested by [Morusya](https://osu.ppy.sh/users/13681464)
+- [cassie - me & u (succducc bootleg) (Spectator) \[us\]](https://osu.ppy.sh/beatmapsets/1281611#osu/2662017), suggested by [Morusya](https://osu.ppy.sh/users/13681464)
+- [roer - ViLLAGE OF CHRYSANTHEMUM (Morusya) \[Chrysanth Haven\]](https://osu.ppy.sh/beatmapsets/2257774#osu/4804041), suggested by [SadEgg](https://osu.ppy.sh/users/10278243)
+
+### ![](/wiki/shared/mode/mania.png) osu!mania (lead by [Maxus](https://osu.ppy.sh/users/4335785))
+
+#### Beatmap of the Season — Curators' Pick
+
+[![](/wiki/shared/news/2025-08-22-beatmap-spotlights-summer-2025/2203489.jpg)](https://osu.ppy.sh/beatmapsets/2203489#osu/4793111)
+
+**[technoplanet - Vomit (\_Stan) \[\[7K\] Blocko's Cascade\]](https://osu.ppy.sh/beatmapsets/2203489#osu/4793111)**, created by [Blocko](https://osu.ppy.sh/users/4075092)\
+*primarily suggested and described by [Maxus](https://osu.ppy.sh/users/4335785)*
+
+This season, we are thrilled to unveil a remarkable osu!original created exclusively for the 7K World Cup 2024. Composed by the talented technoplanet and brought to life through the expert mapping of Blocko, one of the community’s finest 7K mappers, this masterpiece stands proudly as our Beatmap of the Season.
+
+Prepare for an unforgettable challenge! This chart delivers a perfect harmony of rice and long notes, combined with unconventional, high-intensity patterns that will keep players engaged from the very first note. It demands physical precision, technical mastery, and impeccable coordination. A true test for anyone aiming to conquer its most intricate sections. As the music builds toward its epic final climax, the map transforms into an immersive journey, each note meticulously emphasizing the song’s story and emotional progression.
+
+We are beyond excited to present this beatmap as the highlight of the season, confident that it will deliver an unparalleled gameplay experience. Join us in celebrating this original creation, and show your support for both the map and its stunning soundtrack, crafted exclusively for osu!
```

Comment:
styling criteria thing

```suggestion
We are beyond excited to present this beatmap as the highlight of the season, confident that it will deliver an unparalleled gameplay experience. Join us in celebrating this original creation, and show your support for both the map and its stunning soundtrack, crafted exclusively for osu!.
```